packages/pluggableWidgets/gallery-native/src/Gallery.tsx

```diff
@@ -37,6 +112,49 @@ export const Gallery = (props: GalleryProps<GalleryStyle>): ReactElement => {
             hasMoreItems={props.datasource.hasMoreItems ?? false}
             itemRenderer={itemRenderer}
             items={props.datasource.items ?? []}
+            filters={useMemo(
```

Comment:
would this be more readable if it is in a separate function?

Reply:
done